boards/st/nucleo_l552ze_q/board.cmake

```diff
@@ -15,13 +15,11 @@ endif()
 
 # keep first
 board_runner_args(stm32cubeprogrammer "--port=swd" "--reset-mode=hw")
+include(${ZEPHYR_BASE}/boards/common/stm32cubeprogrammer.board.cmake)
 
 set_ifndef(BOARD_DEBUG_RUNNER pyocd)
 set_ifndef(BOARD_FLASH_RUNNER pyocd)
-
 board_runner_args(pyocd "--target=stm32l552zetxq")
 
-# keep first
-include(${ZEPHYR_BASE}/boards/common/stm32cubeprogrammer.board.cmake)
```

Comment:
Some time ago, having this line first was sufficient to have `stm32cubeprogrammer` as default runner, this is what is used on other boards and I prefer keeping this board consistent with others.
However, this is the only board making use of `set_ifndef(BOARD_FLASH_RUNNER pyocd)`.
Can you have a check with this line removed ?

Same for the `set_ifndef(BOARD_DEBUG_RUNNER pyocd)`, having it second behind `stm32cubeprogrammer` which can't provide debug should be sufficient.